src/main/js/main_html5.js

```diff
@@ -446,6 +446,30 @@ require("../../../html5-common/js/utils/environment.js");
       }
     };
 
+    /**
+     * Since there are no standards for error codes or names for play promises,
+     * we'll compile a list of errors that represent a user interaction required error.
+     * @private
+     * @method OoyalaVideoWrapper#userInteractionRequired
+     * @param {string} error The error object given by the play promise when it fails
+     * @returns {boolean} True if this error represents a user interaction required error, false otherwise
+     */
+    var userInteractionRequired = function(error) {
+      var userInteractionRequired = false;
+      if (error) {
+        var chromeError = error.name === "NotAllowedError";
+        //Safari throws the error "AbortError" for all play promise failures
+        //so we'll have to treat all of them the same
+        if (!OO.isChrome || chromeError) {
+          //There is no requirement for muted autoplay on Firefox,
+          //so we'll ignore any Firefox play promise errors
```

Comment:
We'll need to add these in once Firefox has some form of muted autoplay requirement. We currently do not know what errors Firefox will be throwing.